### PR TITLE
Email all issue397

### PR DESCRIPTION
**Pull request by thawpt**
created a list of instructor emails and inserted in the href so that all service learning faculty will be emailed. when button is clicked it will redirect to outlook and set the recipients as faculty from the SL faculty table.

only half the email all button was working so we edited the style 

fixes issue #397

### CODE REVIEW

**Comment by BrianRamsay** [comment]
It looks like this was created from the branch with the add past participants change. I will look at this after that one is reviewed and merged.

File: app/templates/main/manageServiceLearningFaculty.html [hunk 1]
```diff
@@ -102,12 +102,21 @@ <h2>Approved</h2>
...
         <button class="btn btn-primary">Send to Registrar</button>
       </div>
-      {% endif %}
+     {% endif %}
       </div>
     </div>
 
 <div id="wrapper">
     <h1 class="text-center pt-1 pb-4">Service-Learning Faculty</h1>
+    <div style="text-align:right; width:100%; padding:0; margin-bottom: -42px; display: inline-block" >
```
Comment (zachneill) [inline app/templates/main/manageServiceLearningFaculty.html]: Change as many of the style tags to their BS5 class equivalents. E.g. p-0 and w-100 are BS classes that should do the same thing. Look at the utilities found here: https://getbootstrap.com/docs/5.0/utilities/spacing/

File: app/templates/events/trackVolunteers.html [hunk 2]
```diff
@@ -39,16 +39,19 @@ <h1 class="text-center">{{event.name}}</h1>
...
   <div class='form-group d-inline-flex'>
-    <a class="btn btn-success" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addVolunteerModal" style="margin:5px;">Add volunteer</a>
+    <a class="btn btn-success" role="button" href="#" data-bs-toggle="modal" data-bs-target="#addVolunteerModal" style="margin:5px;">Add Volunteer</a>
     {% if event.isRecurring and not event.isFirstRecurringEvent%}
-      <a class="btn btn-success" id="pastVolunteers" role="button" data-bs-toggle="modal" style="margin:5px;">Add previous volunteers</a>
-    {% endif %}
+        {% if recurringVolunteers | count == 0  %}
+            {% set disabled = "disabled" %}
+        {% endif %}
+        <button type="button" class="btn btn-success" id="pastVolunteers" data-bs-toggle="modal" data-bs-target= "#addPastVolunteerModal" style="margin:5px;" {{disabled}}>Add Past Volunteers</button>
```
Comment (zachneill) [inline app/templates/events/trackVolunteers.html]: Avoid style tags with margin if you can do a similar bootstrap utility. https://getbootstrap.com/docs/5.0/utilities/spacing/

**Review by zachneill — CHANGES_REQUESTED** [review]
Look into Bootstrap 5 spacing utilities https://getbootstrap.com/docs/5.0/utilities/spacing/

File: app/templates/main/manageServiceLearningFaculty.html (hunk 1 above)
Comment (BrianRamsay) [reply app/templates/main/manageServiceLearningFaculty.html]: One trouble they had is that we'd have to compile our own bootstrap css with negative margin enabled to use their negative margin classes. And we haven't done that yet

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
Are there unpushed commits? It doesn't look like the z-index fix is in.

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**